frontend/src/components/DataTable/Link.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <router-link :to="`/te/tasks/${item['.key']}`">{{ item['.key'] }}</router-link>
```

Comment:
The component if generic, so you should not use `te` path here. Either pass it as a parameter, or make the component TE-specific.